slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -295,4 +297,29 @@ public void updateValidatorEnabledStatus(final Bytes publicKey, final boolean en
   private boolean isEnabled(final Handle handle, final int validatorId) {
     return validatorsDao.isEnabled(handle, validatorId);
   }
+
+  private boolean isNotValidGVR(final Bytes32 genesisValidatorsRoot) {
```

Comment:
Maybe the cache should live with the validator?

Reply:
My initial thought was that the gvr is "unique" with respective to DBSlashingProtection, however, on the second thought, it can live in GVRValidator as well.

Reply:
Agree with @siladu here, I think the cache should live with the GVRValidator

Reply:
done.